src/model/trust.model.ts

```diff
@@ -39,6 +39,9 @@ export interface Trust {
   HISTORICAL_BO?: TrustHistoricalBeneficialOwner[];
   CORPORATES?: TrustCorporate[];
   review_status?: TrustReviewStatus;
+  start_date_day?: string;
+  start_date_month?: string;
+  start_date_year?: string;
```

Comment:
Does the Trust itself need a start date? I thought this PR was only to add start date to Trust legal entity beneficiaries